schedule/autoyast_mini_no_product.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name:           autoyast_mini_no_product
```

Comment:
you can exclude this file too: autoyast_mini_staging  as it is already added: [autoyast_mini_staging](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/schedule/autoyast_mini_staging.yaml)

Reply:
done